src/User/ProfilePage.jsx

```diff
@@ -31,21 +41,44 @@ const ProfilePage = ({ error, isLoading, user }) => {
     return <p>An error has occurred...</p>;
   }
 
-  const { id, username, firstName, lastName, email, occupation } = user || {};
+  const { id, username, firstName, lastName, email, occupation, managerId } =
+    user || {};
+
+  const displayManager = () => {
+    if (managerName === "No_Manager") {
```

Comment:
good start, for maintainability this should probably be a universal constant that you are importing instead of redeclared in several places, make one centralised place for this value to live and import it to other places that need it.